### PR TITLE
Adds server (for localdev)

### PR DESCRIPTION
**Pull request by justinlittman**

### CODE REVIEW

**Comment by aaron-collier** [comment]
@justinlittman I'm just moving this into the backlog for now until we can all meet in person next week.

File: cmd/server/main.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+  http.HandleFunc("/sparql", sparqlHandler)
+
+  address := fmt.Sprintf("%s:%s", get_env("HOST", ""), get_env("PORT", "8080"))
+  log.Printf("Starting server on %s", address)
+
+  http.ListenAndServe(address, nil)
+}
+
+func req_to_body(req *http.Request) string {
```
Comment (aaron-collier) [inline cmd/server/main.go]: @justinlittman - idiomatic Go expects function names to be camel case, so this should be `reqToBody`

File: cmd/server/main.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  body_buf := new(bytes.Buffer)
+  body_buf.ReadFrom(req.Body)
+  return body_buf.String()
+}
+
+func req_to_headers(req *http.Request) map[string]string {
```
Comment (aaron-collier) [inline cmd/server/main.go]: Change to `reqToHeaders`

File: cmd/server/main.go [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  header := make(map[string]string)
+  for key, value := range req.Header {
+    header[key] = value[0]
+  }
+  return header
+}
+
+func get_env(key string, default_value string) string {
```
Comment (aaron-collier) [inline cmd/server/main.go]: Change to getEnv

File: message/client.go [hunk 4]
```diff
@@ -11,17 +11,19 @@ type Client struct {
...
 	region   string
+	ssl      bool
+
 }
 
 // NewClient returns a new Client instance
-func NewClient(endpoint string, topic string, region string) *Client {
-	return &Client{endpoint: endpoint, topic: topic, region: region}
+func NewClient(endpoint string, topic string, region string, ssl bool) *Client {
+	return &Client{endpoint: endpoint, topic: topic, region: region, ssl: ssl}
```
Comment (aaron-collier) [inline message/client.go]: @justinlittman this looks like it may be a valuable change. Could you do a separate PR for this? I'm happy to if you are onto other things.

**Comment by mjgiarlo** [comment]
@justinlittman @aaron-collier Did we discuss using docker containers for this instead?

**Comment by aaron-collier** [comment]
@mjgiarlo it's a bit more complex, and something we plan to discuss next week when we're all on site.

**Comment by aaron-collier** [comment]
@justinlittman can you close this now that we'll no longer be using Go? Lets discuss how to document provide a local dev structure for the python version.

**Comment by mjgiarlo** [comment]
@justinlittman @aaron-collier should we leave the branch hanging around or nuke it? the `Delete branch` button sings to me.

**Comment by justinlittman** [comment]
@mjgiarlo Once we have comparable up and running for python rewrite, I would agree. However, I've been referring to it while writing the python dev server.
